hs_core/tests/api/rest/test_move_or_rename_file_or_folder.py

```diff
@@ -53,27 +53,27 @@ def test_bad_requests(self):
         unzip_url = "/hsapi/resource/%s/functions/move-or-rename/" % self.pid
         response = self.client.post(unzip_url, {
             "source_path": " ",
-            "target_path": "foo"
+            "target_path": "data/contents/foo"

```

Comment:
Wondering why this change. Why not the hander for this endpoint do this fixed prefixing of directory path?